R/pkg/R/utils.R

```diff
@@ -804,7 +807,7 @@ handledCallJMethod <- function(obj, method, ...) {
 
 captureJVMException <- function(e, method) {
   rawmsg <- as.character(e)
-  if (any(grep("^Error in .*?: ", rawmsg))) {
+  if (any(grepl("^Error in .*?: ", rawmsg))) {
```

Comment:
Sort of follow-up to https://github.com/apache/spark/pull/28367

`any` applies to `logical` -> use `grepl`